pset-iterators.js

```diff
@@ -114,7 +127,12 @@ Expected Result Shape: An array of objects (with each object representing a pres
 console.log('Problem 4:')
 
 // Add your code below this line
+const presidentsBetween1850and1900 = presidents.filter((president) => {
+  if(parseInt(president.took_office) >= 1850 && parseInt(president.took_office) <= 1900)
```

Comment:
Interesting, I was under the impression you had to manipulate the string to grab the year; looks like I learned something new here!